src/infection-model/seir.tsx

```diff
@@ -271,31 +284,24 @@ function getCurveProjections(inputs: SimulationInputs & CurveProjectionInputs) {
   );
 
   // index expected population adjustments by day;
-  // start with tomorrow since these are future releases
-  const tomorrow = addDays(Date.now(), 1);
-  const expectedPopulationChanges = Array(numDays).fill(0);
+  const today = Date.now();
+  const expectedPopulationChanges = Array(numDays + 1).fill(0);
   plannedReleases?.forEach(({ date, count }) => {
     // skip incomplete records
     if (!count || date === undefined) {
       return;
     }
-    const dateIndex = differenceInCalendarDays(date, tomorrow);
+    const dateIndex = differenceInCalendarDays(date, today);
     if (dateIndex < expectedPopulationChanges.length) {
       expectedPopulationChanges[dateIndex] -= count;
     }
   });
 
   // initialize the projections with today's data
-  const {
-    currentDayIncarceratedProjections: initialIncarceratedProjections,
-    currentDayStaffProjections: initialStaffProjections,
-  } = getTotalsBreakdown(singleDayState);
-
-  const dailyIncarceratedProjections = [...initialIncarceratedProjections];
-  const dailyStaffProjections = [...initialStaffProjections];
+  updateProjectionDay(0, singleDayState);
 
-  let day = 0;
-  while (day < numDays) {
+  let day = 1;
```

Comment:
There are _a lot_ of change-by-one edits throughout this PR, which makes me wonder what validation we've done to ensure no regression? In particular, I know you just fixed what was basically a glorified off-by-one issue with the model this week so I want to make sure we didn't regress.

Reply:
yes ironically some of that is because I realized I actually missed some off-by-one errors in that previous PR ... I am going to do another round of manual comparison with the current `master` version with a clear head today to double-check all this again